### ISSUE TITLE
copyCodeBlock transforming all html element attributes to lower case

### ISSUE THREAD

**Issue by fatmali** [Pickra/copy-code-block#16]
@Pickra thanks for this awesome addon. I have a weird problem when trying to use camel-case Html attributes, the code displayed by CCB displays all attributes in lower case. An example is trying to display this:

```
 <div className={styles.myClass}>
            Some content in a div
 </div>
```

ends up being displayed like this:

<img width="801" alt="Screen Shot 2019-09-25 at 10 28 40 PM" src="https://user-images.githubusercontent.com/16671674/65633046-daa00080-dfe3-11e9-9885-ee942b303be1.png">

Notice how the `className` attribute changes to `classname` and the value gets enclosed by quotes. Is there any way to disable this automatic formatting?

**Comment by Pickra** [comment]
Hey @fatmali, so glad you like CCB!

I'm happy to help, but I need more information...
- are you using plain CSS, CSS modules, a css in js solution or something else?
- are you using CCB in storybook?
  - what version?
  - [storybook html](https://storybook.js.org/docs/guides/guide-html/), [storybook react](https://storybook.js.org/docs/guides/guide-react/), other?
- **ignore if using storybook**, are you using CCB in...
  - plain JS?
  - react?
  - other?
- **ignore if using storybook**, what build tool are you using...
  - webpack?
  - rollup?
  - other?

**Comment by fatmali** [comment]
@Pickra :
1) Code Block - https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/story-helpers.js#L10. That function acts as a wrapper for all our stories.

2) I'm using plain css

3) I'm using CCB in `@storybook/html` where I'm trying to document JSX syntax

**Comment by Pickra** [comment]
Thanks @fatmali 

It looks like you have 2 issues:
- CSS class names not showing up correctly
- CCB's displayed code attribute names getting lowercased 

## CSS class names not showing up correctly
I'm not seeing this issue based on your [buttons-and-links html](https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/buttons-and-links/buttons-and-links.stories.html#L22) and your hosted [storybook page](https://styleguide.openmrs.org/?path=/story/openmrs-styleguide--links-and-buttons), after clicking the `jsx` tab.

<img width="672" alt="Screen Shot 2019-09-26 at 3 15 19 PM" src="https://user-images.githubusercontent.com/5226549/65717953-8000b600-e070-11e9-9413-f3cd694800bf.png">


## CCB's displayed code attribute names getting lowercased 
**TLDR; it's not CCB, it's the `DOMParser`**
Verify it's not CCB...
- cloning the repo
- run `npm i` in the terminal
  - requires node 6.0.0 or higher, npm 3.8.6 or higher
- update the [html example](https://github.com/Pickra/copy-code-block/blob/master/examples/html-example.html#L2) `class` attribute to `className`
- run `npm start` in the terminal
- navigate to `http://localhost:8080/` in the browser

**Verify the `DOMParser` is lowercasing attribute names in your browser console...**

<img width="688" alt="Screen Shot 2019-09-26 at 4 21 10 PM" src="https://user-images.githubusercontent.com/5226549/65722177-b42ca480-e079-11e9-9003-3a360b8422b6.png">

**Comment by fatmali** [comment]
Wow, I didn't realize that. I'll close this now. Thanks @Pickra!
